src/tests/system/tests/test_sssctl.py

```diff
@@ -545,3 +545,292 @@ def test_sssctl__analyze_with_logdir_and_non_root_privileges(client: Client, pro
     assert result_root.rc == 0, "analyze failed as root with --logdir"
     assert result_user.rc == 0, "analyze failed as user with --logdir"
     assert result_root.stdout == result_user.stdout, "Root and user output mismatch with --logdir"
+
+
+@pytest.mark.parametrize(
+    "use_fqdn,case_sensitive",
+    [
+        (False, True),
+        (True, True),
+        (True, False),
+    ],
+    ids=["shortnames", "fqdn", "fqdn_case_insensitive"],
+)
+@pytest.mark.tools
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_sssctl__user_show(client: Client, provider: GenericProvider, use_fqdn: bool, case_sensitive: bool):
+    """
+    :title: sssctl user-show displays user information
+    :setup:
+        1. Add users to provider
+        2. Configure SSSD with fqdn and case_sensitive options
+        3. Start SSSD
+        4. Populate cache
+    :steps:
+        1. Run sssctl user-show for users
+        2. Run sssctl user-show for non-existent user
+    :expectedresults:
+        1. User information is displayed correctly
+        2. Command fails for non-existent user
+    :customerscenario: True
+    """
+    u1 = provider.user("user1").add()
+    u2 = provider.user("CamelCaseUser1").add()
+    client.sssd.domain["use_fully_qualified_names"] = str(use_fqdn)
+    client.sssd.domain["case_sensitive"] = str(case_sensitive)
+    client.sssd.start()
+
+    domain_suffix = "@test" if use_fqdn else ""
+    user1_name = f"{u1.name}{domain_suffix}"
+    user2_name = f"{u2.name}{domain_suffix}"
+
+    # Populate cache
+    client.tools.getent.passwd(user1_name)
+    client.tools.getent.passwd(user2_name)
+
+    result = client.sssctl.user_show(user1_name)
+    assert result.rc == 0
+    assert f"Name: {user1_name}" in result.stdout
+
+    result = client.sssctl.user_show(user2_name)
+    assert result.rc == 0
+    expected_name = user2_name if case_sensitive else user2_name.lower()
+    assert f"Name: {expected_name}" in result.stdout
+
+    # Test case sensitivity for lookups
+    lc_user_name = f"{u2.name.lower()}{domain_suffix}"
+    result = client.sssctl.user_show(lc_user_name)
+    assert result.rc == 0
+    if case_sensitive:
+        assert " is not present in cache" in result.stdout
```

Comment:
I don't think the cache check is really in the scope of the test. I'd like to remove these extra secondary checks. WDYT?